setup.py

```diff
@@ -365,6 +468,8 @@ def setup_package():
                 "-DNRN_ENABLE_RX3D=OFF",  # Never build within CMake
                 "-DNRN_ENABLE_MPI=" + ("ON" if Components.MPI else "OFF"),
                 "-DNRN_ENABLE_MPI_DYNAMIC=" + ("ON" if Components.MPI else "OFF"),
+                "-DNRN_ENABLE_MUSIC=" + ("ON" if Components.MUSIC else "OFF"),
+                "" + (("-DMUSIC_ROOT=" + music_home) if Components.MUSIC else ""),
```

Comment:
this should be passed via  build_wheels.bash ` --cmake-prefix` (`CMAKE_PREFIX_PATH`)